### ISSUE TITLE
Help buttons/links in markdown editor

### ISSUE THREAD

**Issue by ethanmdavidson** [BookStackApp/BookStack#1663]
**Describe the feature you'd like**
On the markdown editor, I would like to have a link to some markdown help. It would also be nice to have some buttons which insert the markdown syntax for you.

Two examples of what I mean:

Gitlab markdown editor
![gitlabMarkdownButtons](https://user-images.githubusercontent.com/1311998/64886843-268e9500-d635-11e9-9594-5298703933c5.PNG)

Github markdown editor
![githubEditor](https://user-images.githubusercontent.com/1311998/64886902-3dcd8280-d635-11e9-9cc5-46d9a1ac0610.png)

**Describe the benefits this feature would bring to BookStack users**
For new users, the buttons would greatly ease the learning curve because the buttons provide a familiar UI, and the documentation would be right at their fingertips. It would also help more advanced users because it can be hard to remember all the different markdown syntax. The [existing keyboard shortcuts](https://www.bookstackapp.com/docs/user/markdown-editor/) are nice, but without a link to the documentation most people will never know they exist.

**Additional context**
This is related to #1661

**Comment by berot3** [comment]
Yes please! For me bookstack is not an option until the WYSIWYG-editor stores the pages as markdown and not as html.
It would be a "work-around" to use the markdown-editor. But than the markdown-way is not usable until at least there are some helper-buttons for editors.

**Comment by Technosoft2000** [comment]
👍 +1 to see this feature in future versions :)

**Comment by Logician724** [comment]
:+1: Really need this feature. Cannot adopt markdown as the default editor for all users if only those proficient in it can use it. Or at least provide the option for different users to use different editors.

**Comment by TheNomad11** [comment]
Yes, please, or more shortcuts at least

**Comment by ssddanbrown** [comment]
> Yes, please, or more shortcuts at least

@TheNomad11 The markdown editor does support a range of shortcuts as listed here: https://www.bookstackapp.com/docs/user/markdown-editor/

**Comment by Zazou49** [comment]
A toolbar will be very usefull for beginner !

**Comment by EmberLightVFX** [comment]
Big +1 on having a toolbar for markdown!

I write markdown way faster than using the WYSIWYG editor and it's very easy to use markdown written somewhere else and just paste it into the editor and bam, you're done.

But I sometimes don't remember specific markdown commands like how to create tables or strike-throughs. I then have to go to https://www.markdownguide.org/ and search there until I find what i was looking for.
Having these added to the toolbar as shortcuts would help a lot for things the user don't remember.

Another reason is that we're multiple people writing on bookstack and having a toolbar would/may avoid switching from MD to WYSIWYG to MD to WYSIWYG etc... depending on who's writing and how good they are with MD.

Looking at the code from the now closed PR https://github.com/BookStackApp/BookStack/pull/3952 it doesn't look like that much extra code to support a feature like this.

**Comment by ssddanbrown** [comment]
BookStack is now [managed on Codeberg](https://codeberg.org/bookstack/bookstack).

As part of this change, we're closing all existing GitHub issues and pull requests.
You'll instead find this issue on Codeberg here: 
https://codeberg.org/bookstack/bookstack/issues/1663
  
Any further discussion for this will now be done on Codeberg.
Therefore this GitHub thread will be locked.
